index.html

```diff
@@ -239,7 +238,11 @@ <h2 id="project_tagline">eiffel-remrem-generate</h2>
 <p>Gives:</p>
 
 <pre>[{"data":{"outcome":{"conclusion":"TIMED_OUT","description":"Compilation timed out."},"persistentLogs":[{"name":"firstLog","uri":"http://myHost.com/firstLog"},{"name":"otherLog","uri":"isbn:0-486-27557-4"}]},"links":{"activityExecution":"aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee1","flowContext":"flowContext","causes":["cause1","cause2"]},"meta":{"domainId":"example.domain","id":"babcdb18-b3c8-46c0-8169-684733bf5ac9","type":"eiffelactivityfinished","version":"0.1.6-PBN2","time":1477033634382,"tags":["tag1","tag2"],"source":{"host":"host","name":"name","uri":"http://java.sun.com/j2se/1.3/","serializer":{"groupId":"G","artifactId":"A","version":"V"}}}}]</pre>
-
+<p><strong>For loading protocol jars other than eiffelsemantics :</strong></p>
+we need to provide the <strong><em>-Djava.ext.dirs="Path to other protocol jars".</em></strong> For example:
+<pre><code> $ java -Djava.ext.dirs="/usr/home/jars/" -jar generate-cli.jar -f eiffeljobfinished.json -t eiffeljobfinished -mp protocoltype
```

Comment:
Isn't it better to have an example path like "/path/to/jars" to make it obvious the user have to provide there own path?